packages/core/src/components/slider/slider.tsx

```diff
@@ -168,7 +168,9 @@ export class TdsSlider {
 
   @Watch('value')
   handleValueUpdate(newValue: string) {
-    this.forceValueUpdate(newValue);
+    this.calculateThumbLeftFromValue(newValue);
```

Comment:
I think this makes a lot of sense. But could it be a breaking change?

Reply:
@mJarsater Nice that it makes sense to you, had some talk about it in the support channels and it could be a breaking change if someone updates the `value` outside and of slider and expects a tdsChange to fire. Likelihood of that is very small and something that they hopefully will notice. But since this more correlates to how all the inputs should work and how other components work it should be updated. We just have con communicate it in a good way.